src/protocols/vnc/settings.h

```diff
@@ -300,6 +300,11 @@ typedef struct guac_vnc_settings {
      */
     int wol_wait_time;
 
+    /**
+    * Whether or not to write the disable remote input message to the remote host
+    */
+    bool disable_remote_input;
```

Comment:
I think this should be described at more of a high level. Rather than describe the low-level impact on which VNC protocol messages are sent, this should describe the high-level effect of the setting on session/server behavior.

Reply:
I hope the following message is ok, "Whether or not to disable the input on the server side."